### DISCUSSION TITLE
Hide or show one command from the menu

### DISCUSSION THREAD

**Discussion by B1ackt34** [forntoh/LcdMenu#279] [Q&A]
Hello, great library that i'm using in my Miner Keeper project.
I would like to hide or show some commands of the menu based on events.
```
ITEM_COMMAND("Show alarms", showAlarmsVoid),
ITEM_COMMAND("Reset alarms", resetAlarmsVoid)
```
I manage different alarms in my project and i would like to show a command "show alarms" and "reset alarms" only if there is an alarm on-going.
Is it possible?

Thank you in advance!

**Comment by forntoh** [comment]
This might help https://lcdmenu.forntoh.dev/overview/menu-screen.html#adding-and-removing-items-dynamically
